package.json

```
@@ -15,82 +15,82 @@
     "analyze": "source-map-explorer 'build/static/js/*.js'"
   },
   "dependencies": {
-    "@entur/alert": "^0.5.11",
-    "@entur/button": "^2.2.1",
-    "@entur/chip": "^0.3.4",
-    "@entur/datepicker": "^0.3.7",
-    "@entur/dropdown": "^0.9.0",
-    "@entur/form": "^2.5.0",
+    "@entur/alert": "^0.5.18",
+    "@entur/button": "^2.2.4",
+    "@entur/chip": "^0.3.9",
+    "@entur/datepicker": "^0.3.14",
+    "@entur/dropdown": "^0.6.0",
```

Ser ut som du har revertert denne, jeg bumpet den i en annen PR.
```suggestion
    "@entur/dropdown": "^0.9.0",
```